docs/diga_api.md

```diff
@@ -114,19 +118,43 @@ This is the base url for the endpoint for a specific insurer.
 There is no central DiGA endpoint for verifying and reimbursing DiGA apps and handling might differ across apis.
 Some apis might also not support billing as of now.
 The `Versandart` field in the mapping file can be used to check this beforehand:
+
 - 1 = api is supported for billing
 - 2 = email (none of the insurances in the mapping file use emails)
 - 3 = post
 
 Although insurance companies use their own apis, they do follow an openapi specification for the request which is listed as [DiGA-YAML-Datei (YAML)](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/digaSP_1_0_05.yaml) at the same [gkv page](https://www.gkv-datenaustausch.de/leistungserbringer/digitale_gesundheitsanwendungen/digitale_gesundheitsanwendungen.jsp).
 This [document](https://github.com/alex-therapeutics/diga-api-client/blob/main/ENDPOINT_STATUS.md) lists which apis are currently working with the diga api client.
 
+The mapping file also contains billing-related information about the insurance companies as well as different IK numbers.
+The following section describes what the different IK numbers are used for as this might not be obvious from the documentation.
+
+### IK numbers
+
+The xml file contains the following rows:
+
+```xml
+<bas:Kostentraegerkennung>109034270</bas:Kostentraegerkennung>
+<bas:IK_des_Rechnungsempfaengers>109034270</bas:IK_des_Rechnungsempfaengers>
+<bas:IK_Abrechnungsstelle>660500345</bas:IK_Abrechnungsstelle>
+```
+
+The values for `Kostentraegerkennung` and `IK_des_Rechnungsempfaengers` are the same in this case (not always true) but the `IK_Abrechnungsstelle` is different.
+The IK number `660500345` is the IK number of the api provider, here bitmarck and the other IK number relates to the insurance company.
+When making requests it is important to use the correct IK numbers in the request header or in the `nutzdaten` field.
+
+| xml field name              | name in this repository  | where to use                                                                            |
+| --------------------------- | ------------------------ | --------------------------------------------------------------------------------------- |
+| Kostentraegerkennung        | insuranceCompanyIKNumber | TODO                                                                                    |
+| IK_des_Rechnungsempfaengers | buyerCompanyCreditorIk   | TODO                                                                                    |
+| IK_Abrechnungsstelle        | clearingCenterIKNumber   | used in the request body as `ikempfaenger` & relevant for the encryption of `nutzdaten` |
+
```

Comment:
I see you are you using your vacation for sensible things @SebastianRehfeldt ;)

Reply:
Of course! But you can see on the Todos that I already called it :) 
Gonna finish the PR when I come back. At least clarified some things around the IK numbers which were initially wrong in this file